hubblestack/files/hubblestack_nova/stat_nova.py

```diff
@@ -276,4 +286,4 @@ def _is_permission_in_limit(max_permission, given_permission):
     if given_x and (not allowed_x):
         return False
 
-    return True
+    return True
```

Comment:
Looks like we lost the trailing newline for some reason. Can you add it back in?

Reply:
sure